dist/init/linux-systemd/caddy@.service

```diff
@@ -0,0 +1,21 @@
+; see `man systemd.unit` for configuration details
+; the man section also explains *specifiers* `%x`
+
+[Unit]
+Description=Caddy HTTP/2 web server %I
+Documentation=https://caddyserver.com/docs
+After=network.target
+
+[Service]
+; run user and group for caddy
+User=%i
+Group=http
+ExecStart=/usr/bin/caddy -agree=true -conf=/etc/caddy/Caddyfile
+Restart=on-failure
+; create a private temp folder that is not shared with other processes
+PrivateTmp=true
+; limit the number of file descriptors, see `man systemd.exec` for more limit settings
+LimitNOFILE=8192
```

Comment:
Can this go higher? It should be cranked as high as possible for production sites. (I admit the recommendation that we print about ulimit -n is too low.)